lib/libutils/isoc/include/malloc.h

```diff
@@ -89,4 +89,7 @@ size_t malloc_get_max_allocated(void);
 size_t malloc_get_allocated(void);
 size_t malloc_get_heap_size(void);
 
+void __malloc_lock(void);
+void __malloc_unlock(void);
```

Comment:
Yes, I suppose. What's important is that `malloc_mu` is in a file compiled with kasan instrumentation enabled in order to get a constructor function.